### PR TITLE
Tooling around the repo

### PR DESCRIPTION
**Pull request by refs**
This adds the tooling around the `ocis-accounts` repository.

### CODE REVIEW

File: .github/settings.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
+---
+repository:
+  name: ocis-accounts
+  description: ':atom_symbol: :busts_in_silhouette: Serve Accounts for oCIS'
+  homepage: https://owncloud.github.io/ocis-graph/
```
Comment (micbar) [inline .github/settings.yml]: ocis-accounts

Comment (refs) [reply .github/settings.yml]: that escaped my grep <3 good catch

File: .github/settings.yml [hunk 2]
```diff
@@ -0,0 +1,98 @@
+---
+repository:
+  name: ocis-accounts
+  description: ':atom_symbol: :busts_in_silhouette: Serve Accounts for oCIS'
```
Comment (tboerger) [inline .github/settings.yml]: Two emojis?

File: Readme.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# ownCloud Infinite Scale: Accounts
```
Comment (tboerger) [inline Readme.md]: It should be named README.md, not Readme.md ;)

File: Readme.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+[![Go Doc](https://godoc.org/github.com/owncloud/ocis-accounts?status.svg)](http://godoc.org/github.com/owncloud/ocis-accounts)
+[![Go Report](http://goreportcard.com/badge/github.com/owncloud/ocis-accounts)](http://goreportcard.com/report/github.com/owncloud/ocis-accounts)
+[![](https://images.microbadger.com/badges/image/owncloud/ocis-accounts.svg)](http://microbadger.com/images/owncloud/ocis-accounts "Get your own image badge on microbadger.com")
+
+**This project is under heavy development, it's not in a working state yet!**
+
+## Install
+
+You can download prebuilt binaries from the GitHub releases or from our [download mirrors](http://download.owncloud.com/ocis/accounts/). For instructions how to install this on your platform you should take a look at our [documentation](https://owncloud.github.io/ocis-accounts/)
+****
```
Comment (tboerger) [inline Readme.md]: What is this?

File: docker/Dockerfile.linux.amd64 [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+	apk add ca-certificates mailcap && \
+	rm -rf /var/cache/apk/* && \
+	echo 'hosts: files dns' >| /etc/nsswitch.conf
+
+LABEL maintainer="ownCloud GmbH <devops@owncloud.com>" \
+  org.label-schema.name="oCIS Accounts" \
+  org.label-schema.vendor="ownCloud GmbH" \
+  org.label-schema.schema-version="1.0"
+
+EXPOSE 9120 9124
```
Comment (tboerger) [inline docker/Dockerfile.linux.amd64]: No port matching for https://github.com/owncloud/ocis/wiki/Ports?

Comment (refs) [reply docker/Dockerfile.linux.amd64]: Need to be reserved indeed. Was focused on having something rolling 🎢

File: .github/settings.yml (hunk 2 above)
Comment (refs) [reply .github/settings.yml]: too ugly? :p

File: docker/Dockerfile.linux.amd64 (hunk 5 above)
Comment (refs) [reply docker/Dockerfile.linux.amd64]: done 👍

File: .github/settings.yml (hunk 2 above)
Comment (tboerger) [reply .github/settings.yml]: All repos only got one so far, right? :)

Comment (refs) [reply .github/settings.yml]: You're right, sticking to one :)

**Pull request merged**